plugins/modules/aci_fabric_pod_remote_pool.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_fabric_pod_remote_pool
+short_description: Manage Fabric Pod Remote Pool (fabric:ExtSetupP)
+description:
+- Manage Remote Pools on Fabric Pod Subnets.
+options:
+  pod_id:
+    description:
+    - The Pod ID for the Remote Pool.
+    type: int
+    aliases: [ pod ]
+  description:
+    description:
+    - The description for the Remote Pool.
+    type: str
+    aliases: [desc]
+  remote_id:
+    description:
+    - The Identifier for the Remote Pool.
+    type: int
+    aliases: [ id ]
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  remote_pool:
+    description:
+    - The subnet IP address pool for the Remote Pool.
+    - Must be valid IPv4 or IPv6 and include the subnet mask.
```

Comment:
I am not sure that we support v6 for this either.

Reply:
![image](https://github.com/CiscoDevNet/ansible-aci/assets/98932546/7a5a30d5-1c34-4d40-950d-bdad20303894)
Same here? valid v6?

Reply:
However, IPv6 is valid input it doesn't appear that its actually supported. Removed IPv6 from here.